poglink/bot.py

```diff
@@ -29,9 +32,22 @@ def __init__(
         # self.bans_url = bans_url
         self.data_dir = data_dir
 
+        # handle special cases for list parsing
+        for val in LIST_VALUES:
+            if isinstance(getattr(self, val), str):
+                try:
+                    setattr(self, val, parse_list(getattr(self, val)))
+                except TypeError as e:
+                    logger.warning(
+                        f"Incorrect variable format for {val}; should be comma separated list. Actual value: {config[val]}; {e}"
+                    )
```

Comment:
Direct copy-paste of where this is being used in `setup_config()`. Python is pretty forgiving, so I think it's best to allow unexpected values here but warn about it.